pkg/cloudprovider/provider/openstack/helper.go

```diff
@@ -302,7 +302,7 @@ func getFreeFloatingIPs(client *gophercloud.ProviderClient, region string, float
 
 	var freeFIPs []osfloatingips.FloatingIP
 	for _, f := range allFIPs {
-		if f.Status == "ACTIVE" && f.PortID == "" {
+		if f.Status == "DOWN" && f.PortID == "" {
```

Comment:
I don't know the correct behaviour, I trust you ;) but I would love to see the different states as a `const` - that would be amazing